feature-libs/product-configurator/rulebased/cpq/occ/cpq-configurator-occ.models.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * An interface representing the models used for the communication with the OCC part of the CPQ configurator
+ */
+export namespace OccCpqConfigurator {
```

Comment:
[minor] This namespace is not exported in public API. Was it intentional?

Reply:
Thanks (no, this was not intentional). Will fix